spec/quality/reek_source_spec.rb

```diff
@@ -1,4 +1,5 @@
-require_relative '../spec_helper'
```

Comment:
Isn't requiring the spec_helper kind of best practice even if it's technically not necessary?

I mean, we require files in /lib explicitly for the same reason even though quite a few times it wouldnt be necessary...